api/src/main/java/org/apache/iceberg/OverwriteFiles.java

```diff
@@ -145,4 +151,50 @@
    */
   @Deprecated
   OverwriteFiles validateNoConflictingAppends(Long readSnapshotId, Expression conflictDetectionFilter);
```

Comment:
Can you remove this as well? Looks like it was supposed to be removed already.

Reply:
I'll clean up all interfaces in the api module in a separate PR.